cmd/golangBuild.go

```diff
@@ -106,6 +108,12 @@ func runGolangBuild(config *golangBuildOptions, telemetryData *telemetry.CustomD
 		}
 	}
 
+	if config.CreateBOM {
+		if err := utils.RunExecutable("go", "install", golangCycloneDXPackage); err != nil {
```

Comment:
according to the project page https://github.com/CycloneDX/cyclonedx-gomod#from-source:

> Building from source requires Go 1.17 or newer.

How shall we deal with incompatibility issues here? Use the compiled binary instead or accept this limitation for now?

Reply:
What is the minimum version of golang we should support? If we will use compiled binary, where to store it?